sqs/endpoint.bal

```diff
@@ -224,19 +191,11 @@ public isolated client class Client {
             map<string> parameters = {};
             parameters[PAYLOAD_PARAM_ACTION] = ACTION_DELETE_MESSAGE;
             parameters[PAYLOAD_PARAM_RECEIPT_HANDLE] = receiptHandleEncoded;
-            http:Request|error request = self.generatePOSTRequest(amzTarget, queueResourcePath, 
+            http:Request request = check self.generatePOSTRequest(amzTarget, queueResourcePath, 
                                          self.buildPayload(parameters));
-            if (request is http:Request) {
-                http:Response|error httpResponse = self.clientEp->post(queueResourcePath, request);
-                xml|ResponseHandleFailed response = handleResponse(httpResponse);
-                if (response is xml) {
-                    return xmlToDeleteMessageResponse(response);
-                } else {
-                    return error OperationError(OPERATION_ERROR_MSG, response);
```

Comment:
Should we remove the method `OperationError ` in that case?

Reply:
It is the Distinct Error we created with custom error messages. As we used `check` statement we can throw the exact error thrown

Reply:
I was under impression that `OperationError ` is a normal method in Ballerina. If not, fine.